code/game/objects/items/tools/wrench.dm

```diff
@@ -90,30 +90,16 @@
 	AddElement(/datum/element/update_icon_updates_onmob)
 
 /obj/item/wrench/combat/attack_self(mob/living/user)
-	if(on)
-		on = FALSE
-		force = initial(force)
-		w_class = initial(w_class)
-		throwforce = initial(throwforce)
-		tool_behaviour = initial(tool_behaviour)
-		toolspeed = initial(toolspeed)
-		playsound(user, 'sound/weapons/saberoff.ogg', 5, TRUE)
-		to_chat(user, "<span class='warning'>[src] can now be kept at bay.</span>")
-	else
-		on = TRUE
-		force = 6
-		w_class = WEIGHT_CLASS_NORMAL
-		throwforce = 8
-		tool_behaviour = TOOL_WRENCH
-		toolspeed = 1
-		playsound(user, 'sound/weapons/saberon.ogg', 5, TRUE)
-		to_chat(user, "<span class='warning'>[src] is now active. Woe onto your enemies!</span>")
+	on = !on
+	force = on ? 8 : initial(force)

```

Comment:
Oh, wait, energy wrench is actually used somewhere? I did it because I thought it's not used anywhere, so engies on cruiser will have some sort of a gimmick weapon.